api/types.go

```diff
@@ -608,14 +682,21 @@ func (pins PinSerial) ToPin() Pin {
 	if err != nil {
 		logger.Error(pins.Cid, err)
 	}
+	child, err := cid.Decode(pins.Child)
+	if err != nil {
+		logger.Error(pins.Child, err)
```

Comment:
This is potentially going to complain all the time with pins which rightfully don't have a child. Should just not try to decode if child is nil.